[2013-06-10 05:17:11] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1428
Stack trace:
#0 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/myapp/bootstrap/compiled.php(481): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/myapp/bootstrap/compiled.php(470): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/myapp/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-06-10 06:13:50] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/myapp/bootstrap/compiled.php:541
Stack trace:
#0 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(177): Illuminate\Foundation\Application->abort(404)
#1 /home/gristech/myapp/app/controllers/BlogController.php(177): Illuminate\Support\Facades\Facade::__callStatic('abort', Array)
#2 /home/gristech/myapp/app/controllers/BlogController.php(177): Illuminate\Support\Facades\App::abort(404)
#3 [internal function]: BlogController->getView('this-is-it')
#4 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#5 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#6 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('this-is-it')
#8 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#9 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#10 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /home/gristech/myapp/bootstrap/compiled.php(481): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/myapp/bootstrap/compiled.php(470): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /home/gristech/myapp/public/index.php(49): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-06-10 13:27:40] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/myapp/bootstrap/compiled.php:541
Stack trace:
#0 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(177): Illuminate\Foundation\Application->abort(404)
#1 /home/gristech/myapp/app/controllers/BlogController.php(177): Illuminate\Support\Facades\Facade::__callStatic('abort', Array)
#2 /home/gristech/myapp/app/controllers/BlogController.php(177): Illuminate\Support\Facades\App::abort(404)
#3 [internal function]: BlogController->getView('this-is-it')
#4 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#5 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#6 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('this-is-it')
#8 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#9 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#10 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /home/gristech/myapp/bootstrap/compiled.php(481): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/myapp/bootstrap/compiled.php(470): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /home/gristech/myapp/public/index.php(49): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-06-10 18:06:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1428
Stack trace:
#0 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/myapp/bootstrap/compiled.php(481): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/myapp/bootstrap/compiled.php(470): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/myapp/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-06-10 19:42:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1428
Stack trace:
#0 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/myapp/bootstrap/compiled.php(481): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/myapp/bootstrap/compiled.php(470): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/myapp/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2013-06-10 20:18:19] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/myapp/bootstrap/compiled.php:541
Stack trace:
#0 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php(177): Illuminate\Foundation\Application->abort(404)
#1 /home/gristech/myapp/app/controllers/BlogController.php(177): Illuminate\Support\Facades\Facade::__callStatic('abort', Array)
#2 /home/gristech/myapp/app/controllers/BlogController.php(177): Illuminate\Support\Facades\App::abort(404)
#3 [internal function]: BlogController->getView('this-is-it')
#4 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(138): call_user_func_array(Array, Array)
#5 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Controllers/Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('getView', Array)
#6 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'getView', Array)
#7 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('this-is-it')
#8 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#9 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#10 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /home/gristech/myapp/bootstrap/compiled.php(481): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /home/gristech/myapp/bootstrap/compiled.php(470): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /home/gristech/myapp/public/index.php(49): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2013-06-10 21:22:49] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1428
Stack trace:
#0 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /home/gristech/myapp/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /home/gristech/myapp/bootstrap/compiled.php(481): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /home/gristech/myapp/bootstrap/compiled.php(470): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /home/gristech/myapp/public/index.php(49): Illuminate\Foundation\Application->run()
#5 {main} [] []
